english/plow/plow.txt

```diff
@@ -0,0 +1,6 @@
+Mac: Great, onto the matter at hand. We're getting plowed in the ass by the oil companies and the gas companies with their ten gallon hats and their rotten ass-plowing hearts. So, as the brains of this organization, I came up with a plan.
```

Comment:
It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.